README.md

```diff
@@ -14,15 +14,16 @@ _**Note:** No data, public or private are shared in this repository._
 - The main code is found in the root of the repository (see Usage below for more information)
 - The accompanying [report](./reports/report.pdf) is also available in the `reports` folder
 - More information about the VAE with Differential Privacy can be found in the [model card](./model_card.md)
+- scratch_vae_expts.py is similar to the files you will find within the investigations folder. To re-run our results then scratch_vae_expts.py is all you require. If you want an easy way to understand our code and work process then using the respective notebooks within the investigations folder helps to run through the work.
```

Comment:
Any mention of file paths need wrapping in \` e.g. scratch_vae_expts.py -> `scratch_vae_expts.py`